src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js

```diff
@@ -10,7 +10,7 @@ export const item = Object.assign(
     format(g) {
       const guesstimateType = this.guesstimateType(g)
       const [low, high] = this._numbers(g.text)
-      return {guesstimateType, low, high }
```

Comment:
Precision is probably poorly named, this just specifies the number of nonzero digits in scientific notation to show for types of estimates. So this actually matches current behavior afaict (e.g. [0.00001, 0.00002] has the same # of nonzero digits as [5,7])